tests/app/service/test_rest.py

```diff
@@ -341,3 +342,27 @@ def test_update_service_should_404_if_id_is_invalid(notify_api, notify_db):
                 headers=[('Content-Type', 'application/json'), auth_header]
             )
             assert resp.status_code == 404
+
+
+def test_get_users_by_service(notify_api, notify_db, notify_db_session, sample_user):
+    with notify_api.test_request_context():
+        with notify_api.test_client() as client:
+
+            sample_service = create_service(notify_db=notify_db, notify_db_session=notify_db_session,
+                                            service_name='Sample service', user=sample_user)
+            auth_header = create_authorization_header(
+                path='/service/{}/users'.format(sample_service.id),
+                method='GET'
+            )
+
+            resp = client.get(
+                '/service/{}/users'.format(sample_service.id),
```

Comment:
I'm not 100% on the benefit of url for

- using it means if I change the method sig I change the tests
- not using it means I change the path I change the tests

By all means use it in application code - the right answer - using it in tests to save typing is not 100% benefit.